### ISSUE TITLE
Git operation hangs due to empty string in $GIT_ASKPASS

### ISSUE THREAD

**Issue by lszomoru** [microsoft/vscode#136837]
Copied from https://github.com/microsoft/vscode/issues/121231#issuecomment-964597959

>  Here's what I found was happening on my machine:
```
$ cat $GIT_ASKPASS
#!/bin/sh
VSCODE_GIT_ASKPASS_PIPE=`mktemp`
ELECTRON_RUN_AS_NODE="1" VSCODE_GIT_ASKPASS_PIPE="$VSCODE_GIT_ASKPASS_PIPE" "$VSCODE_GIT_ASKPASS_NODE" "$VSCODE_GIT_ASKPASS_EXTRA_ARGS" "$VSCODE_GIT_ASKPASS_MAIN" $*
cat $VSCODE_GIT_ASKPASS_PIPE
rm $VSCODE_GIT_ASKPASS_PIPE

$ export | grep VSCODE_GIT_ASKPASS_EXTRA_ARGS
declare -x VSCODE_GIT_ASKPASS_EXTRA_ARGS=""
```

> This means that $GIT_ASKPASS is passing the empty string as the first positional argument to node, which causes it to launch a REPL instead of running the script $VSCODE_GIT_ASKPASS_MAIN.
>My workaround is export VSCODE_GIT_ASKPASS_EXTRA_ARGS="--title=foobar" but the proper bugfix should happen in VSCode's GIT_ASKPASS implementation.
> This bug appears to have been introduced by this security bugfix: ac29daa

**Comment by Pixelao** [comment]
I experience the same problem, from the moment ac29daa been released. 
Working through remote ssh the git processes stay open, without anything happening.

**Comment by arifszn** [comment]
Still facing this issue on WSL.

**Comment by gjsjohnmurray** [comment]
Will be fixed in upcoming 1.62.2. Meanwhile a workaround is to [disable auto-updates](https://code.visualstudio.com/docs/supporting/faq#_how-do-i-opt-out-of-vs-code-autoupdates) and then manually [downgrade to 1.62](https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions)

**Comment by phil-opp** [comment]
I can confirm that this is fixed after updating to 1.62.2 (https://github.com/microsoft/vscode/issues/136841).

**Comment by kliakos** [comment]
I am on 1.62.2 and the problem is still here.
